src/collective/cover/testing.py

```diff
@@ -142,6 +142,9 @@ def setUpZope(self, app, configurationContext):
                 self.loadZCML(package=Products.PloneFormGen)
                 z2.installProduct(app, 'Products.PloneFormGen')
 
+            import plone4.csrffixes
```

Comment:
@rodfersou plone4.csrffixes have some backport of Plone 5 csrf stuff. Example: https://github.com/plone/plone.app.imaging/blob/beb2b6ddd35d6003f64bc2673863dafaa0f7892c/src/plone/app/imaging/traverse.py#L66 and https://github.com/plone/plone4.csrffixes/blob/680703ef9dc478c61a49000f4789a55ee52eb12e/plone4/csrffixes/patches.py#L11

In this snippet, you'll need a conditional import to support Plone 4 and 5 at the same time.